app/controllers/manager/reports_controller.rb

```diff
@@ -7,7 +7,7 @@ def index; end
 
   def update
     Report.by_ids(params[:report_ids])
-          .find_each(&:checked!)
+          .find_each{|u| u.update status: params[:update_status]}
```

Comment:
cái này có bị n+1 ko e

Reply:
ban đầu em có dùng update_all nhưng update_all không chạy qua validate nên rubocop nó báo lỗi.
Không có n+1 anh ạ.

Reply:
sao k dùng update_all e

Reply:
update_all rubocop nó báo lỗi "Avoid using update_all because it skips validations" chị ạ

Reply:
disable nó đi e
cho c dùng update_all e nhé

Reply:
Vâng ạ. Để em sửa lại ạ